{"name": "Проверка авторизации курьера с некорректным логином", "status": "passed", "description": "Тест проверяет, что при авторизации курьера с некорректным логином запрос возвращает статус 404 и сообщение об ошибке", "start": 1703771674268, "stop": 1703771674623, "uuid": "2aabc7b8-b168-4395-8151-a99099009c98", "historyId": "6643aadc7ff7bffe78cbb4efba5631a7", "testCaseId": "6643aadc7ff7bffe78cbb4efba5631a7", "fullName": "tests.test_login_courier.TestCourierLogin#test_courier_authorization_with_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-7U5QA71"}, {"name": "thread", "value": "1988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}